modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -39,8 +39,8 @@ To define index patterns and create visualizations in Kibana:
 
 . Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
-** Users must manually create index patterns to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs. 
+** Each user must manually create index patterns when logging into Kibana the first time in order to see logs for their projects. Users should create a new index pattern named *app* and use the `@timestamp` time field to view their container logs.
 
-** Admin users must create index patterns for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 
+** Each Admin user must create index patterns when logged into Kibana the first time for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 
```

Comment:
```suggestion
** Each admin user must create index patterns when logged into Kibana the first time for the `app`, `infra`, and `audit` indices using the `@timestamp` time field. 
```

Reply:
jbrigman completed further suggested modifications for BZ1894835: removing boldface inappropriately used.